src/VisualStudio/Core/Def/Implementation/PullMemberUp/SymbolDependentsBuilder.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.PullMemberUp
+{
+    internal static class SymbolDependentsBuilder
+    {
+        internal static ImmutableDictionary<ISymbol, Task<ImmutableArray<ISymbol>>> FindMemberToDependentsMap(
```

Comment:
nit: in general we would make this a public method inside an internal class.  That's basically just for consistency with the rest of hte IDE.